server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1511,6 +1511,7 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
             lastSentMSN: this.lastSentMSN,
             nackMessages: Array.from(this.nackMessages),
             successfullyStartedLambdas: this.successfullyStartedLambdas,
+            checkpointTimestampEpoch: Date.now(),
```

Comment:
>checkpointTimestampEpoch [](http://example.com/codeflow?start=12&length=24)

(nit) remove Epoch from the name? since we the timestamp implementation doesn't matter in this context.

Reply:
Updated.